TACC/Core-Portal
# Task/FP-1171: Toggle data files [A2CPS]

## Overview: ##
Adds settings value in `settings_custom.py` to disable **Data Files**.

## Related Jira tickets: ##

* [FP-1171](https://jira.tacc.utexas.edu/browse/FP-1171)

## Summary of Changes: ##
1. Added a setting in `_WORKBENCH_SETTINGS` to toggle access to **Data Files**

## Testing Steps: ##
1. Set `hideDataFiles` in `_WORKBENCH_SETTINGS` in `settings_custom.py` to `True`
2. Ensure that access to Data Files is disabled from the sidebar.
3. Ensure that access to Data Files is disabled from site search (except for public-data).
4. Ensure that access to Data Files is disabled everywhere else.

## UI Photos:
N/A

## Notes: ##
Please let me know if I've missed any other places.
The `/public-data` route and related components are not affected in the scope of this PR because it (the public data route) cannot access workbench settings...
